### PR TITLE
manifest: update OpenThread libraries to commit `ce77ab3c`

### PR DESCRIPTION
**Pull request by edmont**
Regular OpenThread libraries update.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

### CODE REVIEW

**Comment by edmont** [comment]
Notice the libraries are not in their final state yet, waiting for https://github.com/nrfconnect/sdk-nrf/pull/6819 to be merged in order to update them. That's why I'm not linking to an actual nrfxlib PR yet. This should help to anticipate any possible issues shown by the CI.

**Comment by shantha-14** [comment]
removing milestone as the dependant  #6819  does not have milestone

File: west.yml [hunk 1]
```diff
@@ -108,7 +108,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: a13cd152d7b70e90fe8dd9bca17d40a2a678f3e9
+      revision: pull/654/head
```
Comment (frkv) [inline west.yml]: This commit needs to be dropped before final  run

**Review by frkv — APPROVED** [review]
Can confirm that this PR contains everything in #6819 and the dependent sdk-nrfxlib pr 654

**Review by mariuszpos — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Review by torsteingrindvik — APPROVED** [review]

**Pull request merged**